systemtests/src/main/java/com/github/streamshub/systemtests/setup/StrimziOperatorSetup.java

```diff
@@ -233,10 +222,11 @@ private List<File> fetchStrimziYamlFiles() {
             LOGGER.debug("Fetching Strimzi YAML file: {}", fileName);
             Path tempFile = null;
             try {
-                tempFile = Files.createTempFile(fileName + "-tmp_", ".yaml");
-                Files.write(tempFile, SetupUtils.getYamlFileContent(fileUrl).getBytes());
+                FileAttribute<Set<PosixFilePermission>> attr = PosixFilePermissions.asFileAttribute(PosixFilePermissions.fromString("rwx------"));
+                tempFile = Files.createTempFile(fileName + "-tmp_", ".yaml", attr);
+                Files.write(tempFile, SetupUtils.getYamlContentFromUrl(fileUrl).getBytes());
```

Comment:
I would create method for this, as it seems you are using this on few places.

Reply:
Sure, can do